index.html

```diff
@@ -2022,7 +2022,19 @@
       <h2>Security Considerations</h2>
 
       <p>
+There are a number of security considerations that <a>issuers</a>,
+<a>holders</a>, and <a>verifiers</a> should be aware of when processing
+data described by this specification. Ignoring or not understanding the
+implications of this section can result in security vulnerabilities to
+the implementer as well as other parts of the ecosystem.
```

Comment:
`can result in security vulnerabilities.`

strike "to the implementer as well as other parts of the ecosystem" -- it seems clunky and unnecessary.

Reply:
Done.